src/DataGenerator.cpp

```diff
@@ -0,0 +1,18 @@
+/******************************************************/
```

Comment:
Same comment as below about generated build files.

Reply:
Changed extension of .ino files to .cpp files.